From 216ac69660ca5a1c9168ce986e69aad2c243781b Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Mon, 14 Nov 2016 15:47:29 -0800
Subject: [PATCH] MANTA-3029 use git+https url in marlin README for "npm
 install" Reviewed by: Jordan Hendricks <jordan.hendricks@joyent.com> Approved
 by: Jordan Hendricks <jordan.hendricks@joyent.com>

---
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.md b/README.md
index c3e7047..33a068f 100644
--- a/README.md
+++ b/README.md
@@ -39,7 +39,7 @@ bits to build on OS X.  To use these tools:
 
 or just:
 
-    $ npm install git+ssh://git@github.com:joyent/manta-marlin.git
+    $ npm install git+https://github.com/joyent/manta-marlin.git
 
 At this point, you'll have several tools available inside the "sbin" directory:
 
-- 
2.21.0

